### DISCUSSION TITLE
Mending tool constantly going to my selected slot.

### DISCUSSION THREAD

**Discussion by Beppe0515** [maruohon/tweakeroo#426] [Q&A]
I think that an item constantly going to my selected slot because it has mending on it, is a feature of this mod. How would i be able (if possible) to disable it. It is very annoying. I have searched some common words such as 'tool' or 'swap', but anything that appeared wasn't its fault for swapping my damaged tools with mending.

**Comment by maruohon** [comment]
Sounds like you have `tweakRepairMode` enabled. It's meant for AFK'ing in an XP farm. (There is also a config in the Lists category for selecting which slots should be used for it.)

**Reply by Beppe0515** [reply]
Ok, thank you so much. I didn't know that you responded so fast!

**Comment by Beppe0515** [comment]
Is it on by default? It's weird that it was active.

**Reply by maruohon** [reply]
No, none of the tweaks are on by default.

**Reply by Beppe0515** [reply]
Ok, Because i don't remember turning it on

**Reply by Beppe0515** [reply]
Anyway thanks for the help

**Reply by maruohon** [reply]
You can also see which configs are at the default values by looking at whether the `RESET` button is active or disabled. If it's disabled, then the config is currently at its default value.

**Reply by Beppe0515** [reply]
Ye, i can see that
